gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -123,17 +122,17 @@
         RetryParams.newBuilder()
             .setRetryBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(100L)
+                    .setInitialDelay(Duration.millis(100L))
                     .setDelayMultiplier(1.2)
-                    .setMaxDelayMillis(1000L)
+                    .setMaxDelay(Duration.millis(1000L))
                     .build())
             .setTimeoutBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(300L)
-                    .setDelayMultiplier(1.3)
-                    .setMaxDelayMillis(3000L)
+                    .setInitialDelay(Duration.millis(2000L))
```

Comment:
Honestly, I am not sure how "back-off" and "max-delay" relates to timeout. Do you mean
initial time before timeout happens is 200L after that further tries are multiplied by 1.5 but will
not exceed 30000L? If so, though I understand that settings may feel similar to retry backoff
the method names do not.